clientgui/AdvancedFrame.cpp

```diff
@@ -195,7 +195,9 @@ BEGIN_EVENT_TABLE (CAdvancedFrame, CBOINCBaseFrame)
     EVT_MENU(ID_HELPBOINCMANAGER, CAdvancedFrame::OnHelpBOINC)
     EVT_MENU(ID_HELPBOINCWEBSITE, CAdvancedFrame::OnHelpBOINC)
     EVT_MENU(wxID_ABOUT, CAdvancedFrame::OnHelpAbout)
+#ifdef ENABLE_NEW_VERSION_CHECK
```

Comment:
As I told you before, there are two additional OSs we support: Windows and OSX. If you build BOINC Manager for linux to control BOINC client for linux - this works perfect. But if you decide to control BOINC client for Windows or OSX with BOINC Manager for Linux - then you'll have no way to check for a new version of BOINC client for that particular OS. The same if applicable for BOINC Manager for Windows or OSX, I'll have an option to check for a new version that will do basically nothing because this functionality just dropped from the BOINC client for linux.